build.rs

```diff
@@ -197,6 +197,11 @@ const ASM_TARGETS: &[AsmTarget] = &[
         arch: AARCH64,
         perlasm_format: "ios64",
     },
+    AsmTarget {
+        oss: MACOS_ABI,
+        arch: AARCH64,
+        perlasm_format: "visionos64",
```

Comment:
How did you decide on this? Does it not work when we use `perlasm_format: "ios64"`?

In any case, it doesn't make sense for `oss: MACOS_ABI` to be here AND for there to be a different perlasm format. Either we should have an `AsmTarget` that has `oss: "visionos"` or we should add `"visionos"` to MACOS_ABI so that the above AsmTarget is used. RIght?